frontend/src/timeline.module.css

```diff
@@ -0,0 +1,254 @@
+@media screen and (max-width: 549px) {
+    body {
+        padding: 0;
+        margin: 0;
+        font-family: 'Inter';
+    }
+    
+    .arrowText1::after {
+        content: '↑';
+    }
+    
+    .arrowText2::after {
+        content: '↓';
+    }
+
+    .weekdayText {
+        display: none;
+    }
+
+    .mobileTimelineContainer {
+        height: 100vh;
+        width: 19.2vw;
+    }
+    
+}
+
+@media screen and (min-width: 570px) {
+    body {
+        padding: 0;
+        margin: 0;
+        font-family: 'Inter';
+    }
+
+    .timelinePageContainer {
+        color: black;
+        text-align: center;
+    }
+
+    .PCTimelineContainer {
+        color: black;
+        text-align: center;
+    }
+
+    .timelineBarContainer {
+        z-index: 1;
+        padding: 0;
+        margin: 0;
+        display: flex;
+        max-width: 100vw;
+        align-content: center;
+        background-color: pink;
+    }
+
+    .daysContainer {
+        width: 84vw;
+        height: 9vh;
+        border-top: 0.4vh solid gray;
```

Comment:
note that the colour used in the figma is `hsl(0, 0%, 85%)` - quite a bit brighter than `gray`